.github/workflows/release.yml

```
@@ -68,7 +68,7 @@ jobs:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
 
       - name: Create versioned storybook
-        run: STORYBOOK_COMPONENTS_VERSION=$(cat package.json | jq '.version') npm run build:storybook
+        run: STORYBOOK_COMPONENTS_VERSION=$(cat package.json | jq .version) npm run build:storybook
```

Looking at jq doc, it seems `--raw-output` prints a string without jsonifying it (i.e. without quotes): https://stedolan.github.io/jq/manual/
```suggestion
        run: STORYBOOK_COMPONENTS_VERSION=$(cat package.json | jq --raw-output '.version') npm run build:storybook
```

---

Updated; my fault, I was looking at jq-action https://github.com/sergeysova/jq-action instead than jq :/